bookshelf/web/index.php

```diff
@@ -18,13 +18,20 @@
 namespace Google\Cloud\Samples\Bookshelf;
 
 use Google\Cloud\Samples\Bookshelf\DataModel\CloudSql;
+use Google\Cloud\Samples\Bookshelf\DataModel\MongoDb;
 use Google\Cloud\Samples\Bookshelf\FileSystem\CloudStorage;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
 $app = require __DIR__ . '/../src/app.php';
 require __DIR__ . '/../src/controllers.php';
-$app['bookshelf.model'] = new CloudSql();
+
+$db = getenv('BOOKSHELF_DATA_BACKEND');
+if ($db == 'mongodb') {
+    $app['bookshelf.model'] = new MongoDb();
+} else {
+    $app['bookshelf.model'] = new CloudSql();
```

Comment:
I just merged in #21, so we should add a [CloudDatastore](https://github.com/GoogleCloudPlatform/getting-started-php/pull/21/files#diff-e59f036339c905e259516dfbeb2d75edR34) to this option as well, and throw an exception if the type is not valid (i.e. `BOOKSHELF_DATA_BACKEND=takashidb`)